packages/components/src/Modal/Modal.tsx

```diff
@@ -37,14 +39,20 @@ const ModalBackdrop = styled.div`
   background: rgba(0, 0, 0, 0.6);
 `;
 
-const StyledModal = styled(Card)`
+const StyledModal = styled(Card)<{ wide: boolean }>`
   position: relative;
   overflow: inherit;
   z-index: 1;
   max-height: 100%;
   overflow: auto;
   margin-bottom: 0;
   max-width: 35rem;
+
+  ${({ wide }) =>
+    wide &&
+    css`
+      width: 720px;
```

Comment:
Das sollte nur die maximal Verfügbare Fläche einnehmen. Hast du geschaut, ob es dann mobil auch noch etwas Luft zum Rand hat?

```suggestion
      width: 100%;
      max-width: 720px;
```

Reply:
Hat keine Luft zum Rand mobil, aber das ist so ja auch gewollt.

Reply:
Hmm, ich denke so 0.5rem - 1rem zum Rand sollte es haben, willst du das mal testen?